capstone/src/main/webapp/group.html

```diff
@@ -3,13 +3,35 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link
       href="http://fonts.googleapis.com/css?family=Roboto"
       rel="stylesheet"
       type="text/css"
     />
+    <link rel="stylesheet" href="styles/group.css"></link>
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <script src="scripts/post.js"></script>
     <title>Group</title>
   </head>
-  <body></body>
+
+  <body onload="loadPosts()">
+    
+    <div class="group-content">
+
+      <div class="create-post">
+        <form id="post-form" method="POST" action="/post" >
+          <span class="user-img align-vertical"></span>
+
+          <input type="text" id="post-input" class="align-vertical" name="post-input" value="Post a status">
+          <i class="fa-paper-plane" ></i>
```

Comment:
Nit: remove the extra space
```suggestion
          <i class="fa-paper-plane"></i>
```

Reply:
removed